layers/+lang/haskell/README.org

```diff
@@ -93,10 +99,9 @@ checkout this [[https://github.com/kazu-yamamoto/ghc-mod/wiki#known-issues-relat
 
 *** Stack users
 
-Time to time =ghc-mod= is not available via latest Stackage LTS version. So if
-you have problems with calling =stack install ghc-mod=, try to use =stack
-install ghc-mod --resolver lts-3.1= (the last known LTS version that had
-=ghc-mod=). But even if it doesn't work, don't panic, it's easy to install it
+=ghc-mod= should be available via the latest Stackage LTS version. If not, try to use
+=stack install ghc-mod --resolver lts-4.0= (the current LTS which has =ghc-mod=).
```

Comment:
@joehillen I don't agree with you. For example, let's check the `lts-3`. The `ghc-mod` was in `lts-3.1`, but it was removed in `lts-3.2`. So in that case latest `lts-3.x` is not good.